hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/BaseHeapData.java

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
I will probably get back to single HeapData. see https://github.com/hazelcast/hazelcast/pull/10629#discussion_r118892903